bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+const glob = require( 'fast-glob' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+const { getModuleDataFromHeaders } = require( './common' );
+
+exports.options = [];
+
+/**
+ * Command for copying the files required for the standalone plugin.
+ */
+exports.handler = async () => {
+	const pluginsFile = path.join( '.', 'plugins.json' );
+	fs.readFile( pluginsFile, 'utf8', ( err, jsonString ) => {
+		if ( err ) {
+			log(
+				formats.error(
+					`Error reading file from disk: "${ err }"`
+				)
+			);
+		}
+
+		try {
+			const plugins = JSON.parse( jsonString );
+			for ( const moduleDir in plugins ) {
+				const pluginVersion = plugins[ moduleDir ]?.version;
+				const pluginSlug = plugins[ moduleDir ]?.slug;
+
+				if ( ! pluginVersion || ! pluginSlug ) {
+					log(
+						formats.error(
+							'The given module configuration is invalid, the module JSON object is missing the "version" and/or "slug" properties, or they are misspelled.'
+						)
+					);
+					return;
+				}
+
+				try {
+					// Copy module files from the folder.
+					const modulePath = path.join( '.', 'modules/' + moduleDir );
+					try {
+						const buildModulePath = path.join( '.', 'build/' + pluginSlug );
+						// Clean up build module files directory.
+						fs.rmSync( buildModulePath, { force: true, recursive: true } );
+
+						fs.copySync( modulePath, buildModulePath, { overwrite: true } );
+					} catch ( copyError ) {
+						log(
+							formats.error(
+								`Error copying plugin file: "${ copyError }"`
+							)
+						);
+						continue;
+					}
+
+					// Update file content.
+					updatePluginHeader( {
+						key: moduleDir,
+						slug: pluginSlug,
+						version: pluginVersion,
+						pluginModulePath: modulePath,
+					} );
+
+					// Update text domain.
+					updateTextDomain( {
+						slug: pluginSlug,
+					} );
+				} catch ( error ) {
+					log(
+						formats.error(
+							`${ error }`
+						)
+					);
+				}
+			}
+		} catch ( jsonError ) {
+			log(
+				formats.error(
+					`Error parsing JSON string: "${ jsonError }"`
+				)
+			);
+		}
+	} );
+};
+
+/**
+ * Updates the `load.php` file header information.
+ *
+ * @param {[]Settings} settings Plugin settings.
+ */
+async function updatePluginHeader( settings ) {
+	const { key, version, slug, pluginModulePath } = settings;
+	const loadFile = path.join( '.', 'load.php' );
+	const loadFileContent = fs.readFileSync( loadFile, 'utf-8' );
+	const buildLoadFile = path.join( '.', 'build/' + slug + '/load.php' );
+	const buildLoadFileContent = fs.readFileSync( buildLoadFile, 'utf-8' );
```

Comment:
Please use the new plugin path parameter for all of this.

Also, why do we need to use both the original module's `load.php` file and the build plugin's `load.php` file? We just copied over the module to the `build` folder in the function that calls this, so it'll be the same.

This can simply become something like:
```suggestion
	const loadFile = path.join( pluginPath, 'load.php' );
	const loadFileContent = fs.readFileSync( loadFile, 'utf-8' );
```

Reply:
@felixarntz `path.join( '.', 'load.php' )` is use to get the plugin root `load.php` header that we copy for all other header data then we update some with module data.

Reply:
@mukeshpanchal27 Ah, I see now what you mean. For which values is that needed? Couldn't we create the entire plugin header string in this function without using the Performance Lab's plugin header?

Reading the plugin header and then replacing strings inside it is what requires all those additional regexes you're using here, I think it's rather messy. If we generate the entire plugin header string from scratch, we don't need all that and can simplify this logic.

Reply:
Thanks for the feedback. This solves all the problems that I am facing with header replacement and removes much code from the function.